src/tinuous/base.py

```diff
@@ -34,6 +34,16 @@
     "skipped": "incomplete",
     "stale": "incomplete",
     "started": "incomplete",
+    # CircleCI:
```

Comment:
hm, there is no status which would correspond to "failed", or it is just "failed" or "failure" known above?  if so, may be something like
```suggestion
    # Additional ones on CircleCI:
```
to make it a tiny bit more explicit on that.

Reply:
CircleCI uses "failed", which is already listed.